### PR TITLE
Outfit overhaul

### LINKED ISSUE #1081: Loadout items still disappearing into waist pack
#### Description of issue
If your backpack is full, items will be put into your pockets and then belt slot. Items that don't fit in the belt slot will attempt to be put into the waist pack and disappear.

#### Difference between expected and actual behavior
Items that don't fit should be dropped on the cryo chamber, but instead they are put into a belt slot and disappear

#### Steps to reproduce
loadout full of clothes, storage items, and other misc items, (dont forget the waist pack)
pick a job that has lots of storage taken up by job items (office worker, engineer) 
items disappear

#### Length of time in which bug has been known to occur

at least since #967 

#### Issue bingo

- [X] Issue could be reproduced at least once
- [X] Issue could be reproduced by different players
- [X] Issue could be reproduced in multiple rounds
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] [Couldn't find an existing issue about this](https://github.com/foundation-19/foundation-19/issues)

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

On request of Tsuru. Fixes up loadsa bugs and changes up how outfits and loadouts spawns in to be less retarded.
fixes #1081.
FUCK YOU TSURU I WANT MY EMAIL AUTOLOGIN BACK

## Why It's Good For The Game

Only notable change is ID cards automatically going into your PDA. I honestly don't see any reason why someone wouldn't want to do this anyways, so it just saves a couple seconds of time.

## Changelog

:cl:
del: Deleted some unused non-SCP outfits.
qol: ID cards go right into your PDA. Two seconds of effort saved!
qol: Loadout spawning process altered to fit items into slots when possible.
fix: Fixes outfit and loadout bugs (FOR REAL THIS TIME)
code: MASSIVE outfit and job cleanup.
/:cl:

### CODE REVIEW

**Comment by TerryTehCoder** [comment]
CL needs to be updated to reflect the changes as well.

**Comment by aidan-bowman** [comment]
> CL needs to be updated to reflect the changes as well.

I will harvest your organs and sell them on the black market.

**Comment by aidan-bowman** [comment]
wtf why did this close

File: code/modules/modular_computers/computers/subtypes/preset_pda.dm [hunk 1]
```diff
@@ -21,6 +21,16 @@
...
+	var/mob/living/carbon/human/H = get_holder_of_type(src, /mob)
+	if(istype(H))
+		if(H.mind?.initial_email_login)
+			var/datum/computer_file/program/email_client/e_client = hard_drive.find_file_by_name("emailc")
+
+			if(e_client && istype(e_client))
+				e_client.stored_login = H.mind.initial_email_login["login"]
+				e_client.stored_password = H.mind.initial_email_login["password"]
+
+
```
Comment (Tsurupeta) [inline code/modules/modular_computers/computers/subtypes/preset_pda.dm]: no

Comment (aidan-bowman) [reply code/modules/modular_computers/computers/subtypes/preset_pda.dm]: FUCK YOU fine i'll remove it

**Pull request merged**